src/constants/api/index.ts

```diff
@@ -2,6 +2,7 @@ import project from './project';
 import toggle from './toggle';
 import user from './user';
 import member from './member';
+import misc from './misc';
```

Comment:
Codacy has a fix for the issue: [Replace `'./misc'` with `"./misc"`](https://app.codacy.com/gh/FeatureProbe/feature-probe-ui/pullRequest?prid=9905084)
```suggestion
import misc from "./misc";
```